addons/print/tests/test_print_printer.py

```diff
@@ -408,3 +409,18 @@ def test27_user_one_default_group_per_report_type(self):
 
         # Default PDF printer should be unaffected
         self.assert_printer_is_user_default(self.group_upstairs, self.user_alice)
+
+    def test28_abandons_print_if_zero_copies_requested(self):
+        """If zero copies are requested not print should take place."""
+        # Odoo changes logging.INFO to 25 in netsvc.py; it seems the only way
+        # to assert the log level is to reference the module attribute
+        # directly.
+        with self.assertLogs('odoo.addons.print.models.print_printer', level=logging.INFO) as cm:
+            self.printer_default.spool_report(self.printer_default.ids,
+                                              'print.report_test_page',
+                                              copies=0)
+            self.mock_subprocess.Popen.assert_not_called()
+            self.mock_subprocess.Popen.reset_mock()
+        self.assertEqual(
+            cm.output,
+            ['INFO:odoo.addons.print.models.print_printer:Zero copies requested, nothing will be printed.'])
```

Comment:
```suggestion
            ["INFO:odoo.addons.print.models.print_printer:Zero copies requested, nothing will be printed."])
```

Reply:
In general this addon uses single quotes for strings, so it seems best to keep it consistent for now?